packages/language-server/src/__tests__/fixtures/script/attr-tags-for-narrowing/index.marko

```diff
@@ -0,0 +1,5 @@
+<list>
+  <for of=[1, 2, 3]>
+    <@item size="small"/>
```

Comment:
Can we add the `^?` comment here for the `size` attribute so it shows in the type snapshot?